public/docs/_examples/intro/dart/lib/hero_list_component_1.html

```diff
@@ -0,0 +1,7 @@
+<!-- #docregion -->
```

Comment:
I did this before I got word back from you on the named docregions. I'll go through all the current PR's I have out and update them. :)